plugins/inventory/aws_ec2.py

```diff
@@ -57,6 +57,7 @@
               - Spot instances may be persistent and instances may have associated events.
           type: bool
           default: False
+          version_added: '2.8'
```

Comment:
```suggestion
          version_added: '1.1.0'
```
This will be the next release version of the collection.